src/test/java/carracing/RacingTest.java

```diff
@@ -9,14 +11,14 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class CarRacingTest {
+public class RacingTest {
     private CarRacing carRacing;
     private List<Car> cars;
 
     @BeforeEach
     void setUp() {
-        cars = Arrays.asList(new Car("Acar"), new Car("Bcar"),
-                new Car("Ccar"));
+        cars = Arrays.asList(new Car("Acar", 0), new Car("Bcar", 0),
+                new Car("Ccar", 0));
         carRacing = new CarRacing(cars);
     }
 
```

Comment:
바로 아래의(25번 라인부터) 우승자를 찾는 기능에 대한 테스트가 빈약해 보여요. 좀 더 다양한 상황들에 대해서 테스트를 수행해보면 어떨까요?
여러 자동차들이 모두 움직이고 제각각의 위치를 갖고 있다던지.. 여기서 `Car(String, int)` 생성자를 이용해서 우승자를 찾을 당시의 자동차들을 만들어서 사용해보면 좀 더 멋진 테스트를 작성할 수 있을 것 같다는 생각이 드네요!
실제로, 이 테스트는 우승자들을 찾는 기능보다는 `move()`에 대한 테스트처럼 보이는 것 같아요. 테스트를 수행할 때, 아래와 같은 API를 사용해서 테스트를 진행해보면 어떨까요?
```suggestion
assertThat(carRacing.makeWinners()).contains("Acar");
```
(해당 라인-25~31번 라인-에 코멘트가 남겨지지 않아 바로 윗줄에 코멘트 남겨요!)